dotCMS/src/main/java/com/dotcms/rest/api/v1/container/ContainerResource.java

```diff
@@ -0,0 +1,115 @@
+package com.dotcms.rest.api.v1.container;
+
+import com.dotcms.contenttype.model.type.ContentType;
+import com.dotcms.repackage.com.google.common.annotations.VisibleForTesting;
+import com.dotcms.repackage.javax.ws.rs.*;
+import com.dotcms.repackage.javax.ws.rs.core.Context;
+import com.dotcms.repackage.javax.ws.rs.core.MediaType;
+import com.dotcms.repackage.javax.ws.rs.core.Response;
+import com.dotcms.repackage.org.apache.commons.lang.StringUtils;
+import com.dotcms.repackage.org.glassfish.jersey.server.JSONP;
+import com.dotcms.rest.InitDataObject;
+import com.dotcms.rest.ResponseEntityView;
+import com.dotcms.rest.WebResource;
+import com.dotcms.rest.annotation.NoCache;
+import com.dotcms.rest.api.v1.site.SiteHelper;
+import com.dotcms.rest.exception.mapper.ExceptionMapperUtil;
+import com.dotcms.util.I18NUtil;
+import com.dotcms.util.PaginationUtil;
+import com.dotcms.util.pagination.ContainerPaginator;
+import com.dotcms.util.pagination.OrderDirection;
+import com.dotcms.util.pagination.SitePaginator;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.APILocator;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotmarketing.business.APILocator'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)